.drone.yml

```diff
@@ -96,6 +34,61 @@ steps:
       from_secret: docker_password
 
 ---
+kind: pipeline
+name: test
+
+depends_on:
+- publish to docker hub
+
+services:
+- name: search-pg
+  image: postgres:9.6
+  # volumes:
+  # - name: search-pg-v
+  #   path: /var/lib/postgresql/data
```

Comment:
should this also be a temp volume like with snoop?

Reply:
yeah, i would prefer if it was